### PR TITLE
pkg.py - cleanup

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

**Comment by scarabeusiv** [comment]
The first commit looks fine.

File: rpmlint/pkg.py [hunk 1]
```diff
@@ -19,8 +19,9 @@
 except ImportError:
     _magic = None
 import rpm
-from rpmlint.helpers import byte_to_string, print_warning
 
+from .helpers import byte_to_string, print_warning
+from .pkgfile import PkgFile
```
Comment (scarabeusiv) [inline rpmlint/pkg.py]: please either do not use relative imports or switch even other files to use them.

File: rpmlint/checks/DocCheck.py [hunk 2]
```diff
@@ -52,7 +51,7 @@ def _check_doc_file_dependencies(self, pkg):
         Check if docfiles create additional dependencies in the package and
         print a warning if so.
         """
-        files = pkg.files()
+        files = pkg.files
```
Comment (scarabeusiv) [inline rpmlint/checks/DocCheck.py]: does this need separate local variable?

File: rpmlint/checks/DocCheck.py [hunk 3]
```diff
@@ -108,7 +107,7 @@ def _check_huge_docs(self, pkg):
         Check the size of the documentation in the package and print a warning
         if it's more than half of the size of the package.
         """
-        files = pkg.files()
+        files = pkg.files
```
Comment (scarabeusiv) [inline rpmlint/checks/DocCheck.py]: Does this need separate local variable?

File: rpmlint/checks/PostCheck.py [hunk 4]
```diff
@@ -149,7 +148,8 @@ def check_binary(self, pkg):
                     self.output.add_info('W', pkg,
                                          'postin-without-ghost-file-creation', f)
 
-    def check_aux(self, pkg, files, prog, script, tag, prereq):
+    def check_aux(self, pkg, prog, script, tag, prereq):
+        files = pkg.files
```
Comment (scarabeusiv) [inline rpmlint/checks/PostCheck.py]: Does this really need a local variable?

File: rpmlint/checks/DocCheck.py (hunk 2 above)
Comment (mimi1vx) [reply rpmlint/checks/DocCheck.py]: i left local variables in places where they were used in more than 2 cases

**Comment by scarabeusiv** [comment]
Aand this can go in :)

**Pull request merged**